static/scripts/helpers/AlertMessageController.js

```diff
@@ -0,0 +1,128 @@
+const moment = require('moment');
+
+const max = 2;
+
+moment.locale('de'); // set the localization
+
+function getIconTag(status) {
+	switch (status) {
+		case 'danger':
+			return '<i class="fa fa-exclamation-circle text-danger"></i>';
+		case 'info':
+			return '<i class="fa fa-check-circle text-success"></i>';
+		default:
+			return '';
```

Comment:
use info as default

Reply:
Info means that the incident has been solved. 
Because Info is misleading, I'm renaming it in done. Info will be the default